prisma/schema.prisma

```diff
@@ -77,3 +80,54 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+// Inheritance reference:
+// https://www.basedash.com/blog/how-to-model-inheritance-in-prisma
+model Volunteer {
+    user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+    userId String @id
+
+    activities Activity[]
+}
+
+model Administrator {
+    user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+    userId String @id
+
+    manageActivities Activity[]
+}
+
+model Activity {
```

Comment:
1. might be a good idea to place defaults for some of the fields
- isVisible: True

2. capacity should be optional because some events dont have a max capacity. I went to phind it haha, here are some of those events 
- Emergency Response， Virtual Volunteering，Event Support (for large events, such as festivals, concerts, or sports games)
- `phind's response: A flexible capacity can better accommodate these dynamic conditions`